rapids/README.md

```diff
@@ -138,15 +138,16 @@ To use XGBoost4j with Spark 2
 
 ### Step 1. Download dataset for Spark RAPIDS XGBoost application
 
-From [Spark examples](https://github.com/rapidsai/spark-examples) repository
+From [Spark examples](https://github.com/NVIDIA/spark-xgboost-examples/tree/spark-2) repository
```

Comment:
Should it be `.../spark-3`?

Reply:
These examples are for XGBoost4j with Spark 2, pre-the Spark 3 release.  Once Spark 3 became the default branch, we needed to specify spark-2.

Reply:
These examples are for XGBoost4j with Spark 2, pre-the Spark 3 release. Once Spark 3 became the default branch, we needed to specify the spark-2 branch for this example.